### ISSUE TITLE
Model saving/loading is not trivial

### ISSUE THREAD

**Issue by wkirgsn** [wkirgsn/thermal-nn#5]
Add examples for pytorch and tensorflow on how to save and load the custom TNN architecture

**Comment by nMaroulis** [comment]
Hi Wilhelm. Just wanted to catch up and ask if there are any updates regarding the exportation of the model. Thanks

**Comment by wkirgsn** [comment]
Well for pytorch it, indeed, seems trivial
```py
model.save(mdl_file_path)  # save
model = torch.jit.load(mdl_file_path)  # load
```
This just worked in the torch notebook. I will add it there.
For tensorflow I will need to check

**Comment by wkirgsn** [comment]
released v0.1.1 with a demo for model save and load for pytorch and tensorflow
